main/app.py

```diff
@@ -0,0 +1,25 @@
+"""Main module for application."""
+
+from wsgiref.simple_server import make_server
+from pyramid.config import Configurator
+from pyramid.view import view_config
+
+
+@view_config(route_name='hello', renderer='string')
+def hello_world(request):
+    """Print 'Hello-world'."""
+    return 'Hello World'
+
+
+def main():
+    """Run server."""
+    if __name__ == '__main__':
+        config = Configurator()
+        config.add_route('hello', '/')
+        config.scan()
+        app = config.make_wsgi_app()
+        server = make_server('0.0.0.0', 8080, app)
+        server.serve_forever()
+
+
+main()
```

Comment:
It's not inversed. The point is that every time you import some module it's contents are being evaluated. This check ensures that it's the direct run of the module via either `./module.py`, or `python -m module`, or `python ./module.py`. In such case `__name__` var is populated with `'__main__'` value, but it contains the actual name of the module whenever it's being imported from other places via `import` statement.